crates/holochain/src/core/workflow/sys_validation_workflow/types.rs

```diff
@@ -1,17 +1,14 @@
 use super::*;
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
 /// The outcome of sys validation
 pub(crate) enum Outcome {
     /// Moves to app validation
     Accepted,
-    /// Stays in limbo because another DhtOp
-    /// dependency needs to be validated first
-    AwaitingOpDep(AnyDhtHash),
```

Comment:
We don't wait for previous deps to validate? This also wasn't being mapped correctly from source errors. Gone and replaced with MissingDhtDep